src/remodal.js

```diff
@@ -431,6 +422,22 @@
   }
 
   /**
+   * Generates a string separated by dashes and prefixed with NAMESPACE
+   * @private
+   * @param str, str, ...
```

Comment:
Sure, will change that. Are there any docs on how these comments should be generated?